tests/Controller/CountryApiTest.php

```diff
@@ -11,7 +11,6 @@
 
 namespace Sylius\Tests\Controller;
 
-use Lakion\ApiTestCase\JsonApiTestCase;
```

Comment:
Ok, now I get [it](https://github.com/Sylius/Sylius/pull/5513/files#diff-26bb597180cbeccaeb3bf94d02551b01R5).